src/main/frontend/app/containers/FashionItemFormContainer.js

```diff
@@ -0,0 +1,86 @@
+import React, {Component} from 'react';
+
+class FashionItemFormContainer extends Component {
+constructor(props){
+    super(props);
+    this.state = {
+        name: " ",
+        clothingType:" ",
+        styles: " ",
+        bodyType: " " ,
+        measurements:" ",
+        brand:" ",
+        fabricWeight: " ",
+        quality: " ",
+        itemSize:" ",
+        photo: " ",
+        budget:" ",
+        user:" "
+    };
+    this.handleSubmit = this.handleSubmit.bind(this)
+    this.clearForm = this.clearForm.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+}
+
+handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value})
+  }
+handleSubmit(event) {
+    event.preventDefault()
+    this.props.addNewItem(this.state)
+    this.clearForm()
+}
+
+clearForm(){
+    this.setState({
+        name: " ",
+        clothingType:" ",
+        styles: " ",
+        bodyType: " " ,
+        measurements:" ",
+        brand:" ",
+        fabricWeight: " ",
+        quality: " ",
+        itemSize:" ",
+        photo: " ",
+        budget:" ",
+        user:" "
+    })
+}
+
+render() {
+    return (
```

Comment:
you have inconsistent indentation below here, please realign. LMK if you need advice,